server/appbackend/api.py

```diff
@@ -109,76 +115,52 @@ def post(self, request, *args, **kwargs):
 
 class ConfirmTransferRequestApi(generics.UpdateAPIView):
     queryset = TransferRequest.objects.all()
-    serializer_class = TransferRequestSerializer
+    serializer_class = []
 
-    def put(self, request, *args, **kwargs):
+    def patch(self, request, *args, **kwargs):
         transfer_request = self.queryset.get(pk=kwargs["pk"])
         transfer_request.confirm()
         return Response(TransferRequestSerializer(transfer_request).data)
 
+    @swagger_auto_schema(method='put', auto_schema=None)
+    @api_view(['PUT'])
```

Comment:
Shouldn't it be patch and not put in api_view?

Reply:
It is, this suppresses the put view in Swagger, leaving only the patch one.